boot-cw-plus/src/contracts/cw1_subkeys.rs

```diff
@@ -0,0 +1,27 @@
+use boot_core::{contract, Contract, CwEnv};
+use cosmwasm_std::Empty;
+pub use cw1_subkeys::msg::{
+    ExecuteMsgFns as Cw1SubkeysExecuteMsgFns, QueryMsgFns as Cw1SubkeysQueryMsgFns,
+};
+use cw1_whitelist::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
```

Comment:
This should probably be `cw1_subkeys::msg`

Reply:
nice spot